src/core/public/core_app/status/components/metric_tiles.tsx

```diff
@@ -7,24 +7,99 @@
  */
 
 import React, { FunctionComponent } from 'react';
-import { EuiFlexGrid, EuiFlexItem, EuiCard } from '@elastic/eui';
-import { formatNumber, Metric } from '../lib';
+import { EuiFlexGrid, EuiFlexItem, EuiCard, EuiStat } from '@elastic/eui';
+import { DataType, formatNumber, Metric } from '../lib';
 
 /*
- * Displays a metric with the correct format.
+ * Displays metadata for a metric.
  */
-export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
-  const { name } = metric;
+export const MetricCardFooter: FunctionComponent<{
+  testSubjectName: string;
+  title: string;
+  description: string;
+}> = ({ testSubjectName, title, description }) => {
   return (
-    <EuiCard
-      data-test-subj={`serverMetric-${formatMetricId(metric)}`}
-      layout="horizontal"
-      title={formatMetric(metric)}
-      description={name}
+    <EuiStat
+      data-test-subj={testSubjectName}
+      title={title}
+      titleSize="xxs"
+      description={description}
+      reverse
     />
   );
 };
 
+/*
+ * Displays a metric with the correct format.
+ */
+export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
+  const { name } = metric;
+  if (name === 'Delay') {
+    return (
+      <EuiCard
+        data-test-subj={`serverMetric-${formatMetricId(metric)}`}
+        title={formatMetric(metric)}
+        textAlign="left"
+        description={`${name} avg`}
+        footer={
+          metric.meta && (
+            <MetricCardFooter
+              testSubjectName="serverMetricMeta"
```

Comment:
I think we usually try to make `data-test-subj` unique on the page, so maybe we should do something like `serverMetricMeta-${formatMetricId(metric)}` as was done in `EuiCard` above?

Reply:
I had to go with a single `data-test-subj` for the `MetricCardFooter` component, in the same way as we use the static "serverMetric" for the flex items.

Reply:
> I think we usually try to make data-test-subj unique on the page

It depends, we usually do when we need to be able to select each node individually from FTR tests. Some elements have shared testSubj id depending on the needs (e.g find all rows from a table).

So it's probably fine. But just to understand:

> I had to go with a single data-test-subj for the MetricCardFooter component

Why were you forced to do it? Can't we go with `serverMetric-${formatMetricId(metric)}-footer` or something?

Reply:
I wasn't so much _forced_ as wanting to KISS the functional test and use `testSubjects.findAll`.